[DOC] use --variant instead of --mode (#4210)
facebook/react-native-website: docs/running-on-device.md

> If you get a "bridge configuration isn't available" error, see [Using adb reverse](running-on-device.md#method-1-using-adb-reverse-recommended).

> > Hint: You can also use the `React Native CLI` to generate and run a `release` build (e.g. from the root of your project: `yarn android --variant release`).

Reviewer: ```suggestion
> Hint: If you use the `React Native Community CLI`, you can generate a `release` build with `yarn android --mode release`.
```

</TabItem>
</Tabs>

> > Hint: You can also use the `React Native CLI` to generate and run a `release` build (e.g. from the root of your project: `yarn android --variant release`).

Reviewer: same as above

> If you get a "bridge configuration isn't available" error, see [Using adb reverse](running-on-device.md#method-1-using-adb-reverse-recommended).

> > Hint: You can also use the `React Native CLI` to generate and run a `release` build (e.g. from the root of your project: `yarn android --variant release`).

Reviewer: same as above
